src/Linux/Host/Broker/Impl/Services/LinuxRHostProcessService.cs

```diff
@@ -41,15 +45,37 @@ class LinuxRHostProcessService : IRHostProcessService {
             return new UnixProcess(process);
         }
 
+        private string GetUsrLibPath() {
+            const string x64Lib = "/usr/lib/x86_64-linux-gnu";
```

Comment:
For the latter, though, we only need to add the R directory to `LD_LIBRARY_PATH`, right? The standard `/usr/lib` etc entries should already be there - or should be implied.